crypto/getenv.c

```diff
@@ -13,9 +13,45 @@
 
 #include <stdlib.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
 
 char *ossl_safe_getenv(const char *name)
 {
+#if (defined(OPENSSL_SYS_WINDOWS)) && defined(CP_UTF8) && !defined(_WIN32_WCE)
+
+    if (GetEnvironmentVariableW(L"OPENSSL_WIN32_UTF8", NULL, 0) == 0)
+        return getenv(name);
+
+    /* convert name to wide string */
+    int rsize = mbstowcs(NULL, name, 0) + 1;
```

Comment:
These vars such as rsize cant be in the middle of the code..
declare them first at the top..

```
int rsize, fsize;
DWORD envlen;
char *val = NULL;
```

Reply:
I rewrote the code taking into account the comments (_alloca => _malloca, NULL checking, etc.). 
What is the correct procedure not to make a fuss?

Reply:
you need to push the changes so we can see them :)

Reply:
Thanks, done.

Reply:
When you address something, click on the resolve conversation to mark it as done..